QRCoderTests/QRCoderTests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>net35;net452;netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
+		<TargetFrameworks>net5.0-windows</TargetFrameworks>
```

Comment:
Why were the different targets replaced by net5.0-windows? Especially with respect to the different compiler switches where different code is executed in QRCoder for different targets, I think the test cases should be targeted against the same targets as QRCoder main lib.
![image](https://user-images.githubusercontent.com/2236234/116809444-5b1ecf80-ab3e-11eb-936c-9b47d211f593.png)
Or do I oversee something? E.g. if an upcoming change works fine in net5.0, but may break the QRCoder version that targets netstandard1.1, how would we recognize it?

Reply:
This was my mistake I am was already aware of after few tests. I will revert it.

Reply:
Ok, that's fine for me. :-)